includes/amp-helper-functions.php

```diff
@@ -932,6 +932,13 @@ function amp_get_content_sanitizers( $post = null ) {
 		];
 	}
 
+	if ( 'twentytwenty' === get_template() && version_compare( get_bloginfo( 'version' ), '5.3.1', '<' ) ) {
```

Comment:
This isn't technically correct.  It should be checking the version of the theme not core. It's possible for someone to have an older version of Twenty Twenty but have the current version of core.

Reply:
That's what I initially had in mind as well, but [the version code](https://github.com/WordPress/wordpress-develop/blob/388524c939f5efa66010231ba60ff2a8756c75e8/src/wp-content/themes/twentytwenty/style.css#L4) for Twenty Twenty hasn't been bumped since it was integrated into Core.

Reply:
> It's possible for someone to have an older version of Twenty Twenty but have the current version of core.

Hmm. The current version of the theme is `1.0.0`. Would checking for that also suffice?

Reply:
Yeah, if the version is greater than 1.0.0 then the config should be omitted.

Reply:
Version number for Twenty Twenty should be bumped via Trac [ticket #48944](https://core.trac.wordpress.org/ticket/489440), so only checking the theme's version code should be fine, as stated earlier.

Reply:
Done in https://github.com/ampproject/amp-wp/pull/3912/commits/a1528d53e0f95c1274855a2ad9897dfdb3fcc7dd.